spec/features/email_spec.rb

```diff
@@ -27,17 +27,30 @@
     expect(page).to have_text(I18n.t('.emails.submit.sending_successful'))
   end
 
-  scenario "logged in as Organizer after sending an Email to the applicants the event application status gets locked" do
+  scenario "logged in as Organizer after sending an acceptance Email to the applicants the event application status for sending acceptances gets locked" do
```

Comment:
I dont think you should rely on a model test for that, how will you notice then you have to change test this too, if the model changes and there is no reference to `participant_selection_locked`?
However, I didnt see that you changed a feature test 2 screens further down, that checks that one of the two is enough to lock it. So its all fine now :)